src/AssimpWorker/worker/Worker.cpp

```diff
@@ -73,7 +73,8 @@ namespace AssimpWorker {
 	void Worker::importSingleColladaFile() {
 		std::istream& response = storageService.retrieveFile(currentWorkUnit->sourcePath);
 		// Patches welcome...
-		Poco::TemporaryFile tmp;
+		std::string decompressionPath = Configuration::getInstance().get("decompression-path").as<std::string>();
```

Comment:
Good idea in principle. However, I just noticed that this defaults to inside the working directory of the worker. It should default to the system/session temp directory.